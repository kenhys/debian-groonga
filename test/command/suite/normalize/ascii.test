normalize NormalizerAuto "aBcDe 123"
