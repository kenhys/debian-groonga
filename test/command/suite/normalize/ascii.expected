normalize NormalizerAuto "aBcDe 123"
[[0,0.0,0.0],{"normalized":"abcde 123","types":[],"checks":[]}]
